File: test/fixtures/php-pkg/Point.php
<?php

namespace MyApp;

use MyApp\Helper;
use MyApp\Other as OtherThing;

/**
 * Adds two ints.
 */
function add($a, $b) {
    return $a + $b;
}

class Point {
    public $x;
    public $y;

    public function __construct($x, $y) {
        $this->x = $x;
        $this->y = $y;
    }

    public function sum() {
        return $this->x + $this->y;
    }

    private function hidden() {
        return 0;
    }
}

interface Shape {
    public function area();
}

File: test/fixtures/swift-pkg/Point.swift
import Foundation

/// Adds two ints.
func add(a: Int, b: Int) -> Int {
    return a + b
}

private func secret() -> Int {
    return 0
}

class Point {
    var x: Int
    var y: Int

    init(x: Int, y: Int) {
        self.x = x
        self.y = y
    }

    func sum() -> Int {
        return x + y
    }
}

struct Size {
    var w: Int
    var h: Int
}

protocol Shape {
    func area() -> Double
}

enum Color {
    case red, green, blue
}
